tools/extras/install_morfessor.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright  2017  Atlas Guide (Author : Lucas Jo)
+#
+# Apache 2.0
+#
+
+echo "#### installing morfessor"
+dirname=morfessor
+if [ ! -d ./$dirname ]; then
+	mkdir -p ./$dirname
+	git clone https://github.com/aalto-speech/morfessor.git morfessor ||
+		{
+			if [ $? -ne 0 ]; then
+				echo  >&2 "$0: Error git clone operation "
+				echo  >&2 "  Failed in cloning the github repository (https://github.com/aalto-speech/morfessor.git)"
+				echo  >&2 "  Download morfessor-2.0.1.tar.gz instead"
+				wget http://morpho.aalto.fi/projects/morpho/morfessor-2.0.1.tar.gz
+				tar -zxvf morfessor-2.0.1.tar.gz
+				mv Morfessor-2.0.1/ $dirname
+			fi
+		}
+fi
+
+# local installation
+site_packages_dir=$(python -m site --user-site | grep -oE "lib.*")
```

Comment:
typically we try to not to install the packages system-wise (user-wise) -- we keep the libs in kaldi/tools/<package> and just set up the shell variables in env.sh (PYTHONPATH). Otherwise, you could just use pip install morfessor.

Also, in that case, the uninstallation is not needed -- you just delete the directory.

Reply:
@jtrmal 
That line is not intended to install morfessor user-wise.
It is just to grep the name of site-package name and use the name for PYTHONPATH variable,
otherwise, sometime installation process could get an error.

Anyway after installation process, the morfessor libs will be located inside kaldi/tools/